client/src/components/ProTx/components/charts/MaltreatmentTypesPlot.js

```diff
@@ -303,7 +302,6 @@ MaltreatmentTypesPlot.propTypes = {
   mapType: PropTypes.string.isRequired,
   geography: PropTypes.string.isRequired,
   maltreatmentTypes: PropTypes.arrayOf(PropTypes.string).isRequired,
-  observedFeature: PropTypes.string.isRequired,
```

Comment:
This is actually used by the DebugPlot component to render all the available state, so it will break debug mode, but thats fine as I need to refactor that to be a "dumb" component anyway. Just fyi - there is a side effect (that should not be).